src/main/java/hotelreservation/service/LoginService.java

```diff
@@ -0,0 +1,43 @@
+package hotelreservation.service;
```

Comment:
This is fine, but there is a user service already I created which has an authentication function in it you could of used.   It would of reduced the number of services and we could use the same authentication mechanism for both rest and login calls reducing the amount of code.

Reply:
I would of added a new function for browser side authentication that used the "isAuthorized function" to gain access.